tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -501,10 +501,16 @@ export class WebGPUBackend extends KernelBackend {
     };
 
     const kernelMs = await Promise.all(flattenedActiveTimerQueries);
-    res['kernelMs'] = util.sum(kernelMs);
+    res['kernelMs'] = webgpu_util.sum(kernelMs);
```

Comment:
There is no need for a new sum function. Instead, pass "kernelMs.map(item => item[0]);" should be easier.

Reply:
done